src/Document.php

```diff
@@ -11,19 +11,22 @@ class Document extends \DOMDocument {
 
 public function __construct($document = null) {
 	parent::__construct("1.0", "utf-8");
-	$this->registerNodeClass("\\DOMNode", "\\Gt\\Dom\\Node");
-	$this->registerNodeClass("\\DOMElement", "\\Gt\\Dom\\Element");
-	$this->registerNodeClass("\\DOMAttr", "\\Gt\\Dom\\Attr");
-	$this->registerNodeClass("\\DOMDocumentFragment",
-		"\\Gt\\Dom\\DocumentFragment");
-	$this->registerNodeClass("\\DOMDocumentType", "\\Gt\\Dom\\DocumentType");
-	$this->registerNodeClass("\\DOMCharacterData", "\\Gt\\Dom\\CharacterData");
-	$this->registerNodeClass("\\DOMText", "\\Gt\\Dom\\Text");
-	$this->registerNodeClass("\\DOMComment", "\\Gt\\Dom\\Comment");
+	$this->registerNodeClass("\\DOMNode", Node::class);
+	$this->registerNodeClass("\\DOMElement", Element::class);
+	$this->registerNodeClass("\\DOMAttr", Attr::class);
+	$this->registerNodeClass("\\DOMDocumentFragment", DocumentFragment::class);
+	$this->registerNodeClass("\\DOMDocumentType", DocumentType::class);
+	$this->registerNodeClass("\\DOMCharacterData", CharacterData::class);
+	$this->registerNodeClass("\\DOMText", Text::class);
+	$this->registerNodeClass("\\DOMComment", Comment::class);
```

Comment:
Nice cleanup here, but why not use the `::class` notation for the inbuilt classes? It always feels dirty to have a class name in a string.